viz.yaml

```diff
@@ -97,10 +101,20 @@ visualize:
     visualizer: visualizeFishTrends
     mimeType: image/svg+xml
     export: true
+    fig.config:
+        col: '#01b29F'
```

Comment:
hmm trying to be general with these names so that I can use them in the single visualize function. Not sure how I could do a specific name for the walleye color?